deploy/docker/seqr/entrypoint.sh

```diff
@@ -24,9 +24,6 @@ if [ -e "/.config/service-account-key.json" ]; then
     cp /.config/boto /root/.boto
 fi
 
-# link to persistent disk dir with static files
-mkdir -p /seqr_static_files/generated_files
```

Comment:
On this, I'm wondering if we do in fact want to leave something here for local install users that don't use a GCS bucket?

Do you think doing this conditionally if the `STATIC_MEDIA_DIR` env var is set is appropriate? Maybe we should have a convo about what to do about this in the case of local installs -- we probably want to avoid having people accumulate local state in a non-persistent location in the container.

Reply:
I guess if we use something like a volume mount in the docker-compose file, we won't need to do anything in the entrypoint to create this directory for local install users

Reply:
I think using a volume for local installations is probably the way to go here. django will create the nested `generated_files` directory on its own if its needed so with the volume mounted local users wont need any additional set up here